apps/daffio/tslint.json

```diff
@@ -4,13 +4,13 @@
     "directive-selector": [
       true,
       "attribute",
-      "daffodil",
+      "daffio",
       "camelCase"
     ],
     "component-selector": [
       true,
       "element",
-      "daffodil",
+      "daffio",
       "kebab-case"
     ]
```

Comment:
You could make the component name changes I suggested above by adding
```
"component-class-suffix": [
      true,
      "Component",
      "Container"
]
```
here. Or I guess it could probably be added to the global `tslint.json` file, since it already exists in the demo tslint file.

Reply:
resolved in dbcf3deb24b3190d5c086b952e576b6e7fca991d

Reply:
Since you added it in the global tslint, could you remove those same lines from the `demo` and `design` tslint files? Since they aren't needed there anymore

Reply:
@lderrickable I thought about this problem a bit more, I think it's ok for Containers to exist in all `apps`, but not in `libs` (sans state, for the time being). I removed the key in design, I don't know why it was there, and I changed it from global to opt-in for each app.

Reply:
ok sounds good